sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/SearchIndexerSkillsetClient.java

```diff
@@ -76,22 +74,19 @@ public SearchIndexerSkillset getSkillset(String skillsetName) {
      * @return the list of skillsets.
      */
     public PagedIterable<SearchIndexerSkillset> listSkillsets() {
-        return listSkillsets(null, null, Context.NONE);
+        return listSkillsets(null, Context.NONE);
     }
 
     /**
      * Lists all skillsets available for an Azure Cognitive Search service.
      *
-     * @param select selects which top-level properties of the skillset definitions to retrieve. Specified as a
-     * comma-separated list of JSON property names, or '*' for all properties. The default is all properties
      * @param requestOptions additional parameters for the operation. Contains the tracking ID sent with the request to
      * help with debugging
      * @param context additional context that is passed through the HTTP pipeline during the service call
      * @return the list of skillsets.
      */
-    public PagedIterable<SearchIndexerSkillset> listSkillsets(String select, RequestOptions requestOptions,
-        Context context) {
-        return new PagedIterable<>(asyncClient.listSkillsets(select, requestOptions, context));
+    public PagedIterable<SearchIndexerSkillset> listSkillsets(RequestOptions requestOptions, Context context) {
```

Comment:
Only `SearchIndexClient` should return a pageable for its `list` method: #10434

Reply:
I think we have discussed this in chat. Java use Paging even it only returns one page only. @alzimmermsft Please correct me if I am wrong.

Reply:
@sima-zhu you are correct, the general pattern in Java is to use the paging style API for non-paged service responses since it covers dual purposes of being pages but also a list style response.